test/models/vrp_consistency_test.rb

```diff
@@ -571,5 +571,24 @@ def test_pickup_timewindow_after_delivery_timewindow
 
       OptimizerWrapper.wrapper_vrp('demo', { services: { vrp: [:demo] }}, TestHelper.create(problem), nil)
     end
+
+    def test_unicity_of_provided_services_or_vehicles_in_relation
```

Comment:
uniqueness

@fonsecadeline can you activate  `Code Spell Checker`?

Reply:
It is activated but does not work with ruby. I did not find an alternative yet. 
However, unicity would not have been shown because the word does exist according to google translate + reverso.
Still, I will edit this.

Reply:
I don't know why it doesn't work for you, it works for me and it does underline that word. 

Linguistically speaking yes, that word exists; however, we are not linguists, google search `define unicity` specifically mentions that this is a "rare" word and the corresponding french search `define unicité` only gives "uniqueness" as translation.

Reply:
Bizarre... The documentation page specifically lists available languages and ruby is not in it. Are you sure you have no other extension for this ?